### PR TITLE
New vehicle data HandsOffSteering

### PR DESCRIPTION
**Pull request by atiwari9**
This proposal is to add new vehicle data `HandsOffSteering` to know whether or not a driver has his or her hands on the steering wheel.

### CODE REVIEW

**Comment by atiwari9** [comment]
@theresalech  - Ready for review

File: proposals/NNNN-New-vehicle-data-HandsOffSteering.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# New vehicle data HandsOffSteering
+
+* Proposal: [SDL-NNNN](NNNN-New-vehicle-data-HandsOffSteering.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | Policy Server | SHAID | iOS | Java Suite | RPC]
+
+## Introduction
+
+This proposal is to add new vehicle data `HandsOffSteering` to know if driver has hands on steering or not.
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-HandsOffSteering.md]:
```suggestion
This proposal is to add new vehicle data `HandsOffSteering` to know whether or not a driver has his or her hands on the steering wheel.
```

File: proposals/NNNN-New-vehicle-data-HandsOffSteering.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+## Motivation
+
+In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content. Information about whether driver has hands on steering or not is useful for apps that compute driver score and provide insurance.
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-HandsOffSteering.md]:
```suggestion
In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content. Information about whether a driver has hands on the steering wheel or not is useful for apps that compute driver score and provide insurance.
```

File: proposals/NNNN-New-vehicle-data-HandsOffSteering.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+```
+
+## Potential downsides
+
+Author is not aware of any downsides to proposed solution. This proposal just enhances the SDL content.
+
+## Impact on existing code
+
+* SDL Core needs to be updated as per new API.
+* iOS/Java Suite needs to be updated to support getters/setters as per new API.
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-HandsOffSteering.md]:
```suggestion
* iOS/Java Suite need to be updated to support getters/setters as per new API.
```

File: proposals/NNNN-New-vehicle-data-HandsOffSteering.md (hunk 1 above)
Comment (theresalech) [reply proposals/NNNN-New-vehicle-data-HandsOffSteering.md]: Please also make this change in the PR description.

Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-HandsOffSteering.md]: Done

File: proposals/NNNN-New-vehicle-data-HandsOffSteering.md (hunk 3 above)
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-HandsOffSteering.md]: Done

File: proposals/NNNN-New-vehicle-data-HandsOffSteering.md (hunk 2 above)
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-HandsOffSteering.md]: Done

**Comment by atiwari9** [comment]
@theresalech - I have updated the proposal as per your review comments.

**Pull request merged**